charts/cp-kafka/templates/statefulset.yaml

```diff
@@ -138,6 +136,7 @@ spec:
         - |
           export KAFKA_BROKER_ID=$((${HOSTNAME##*-}+${KAFKA_BROKER_ID_OFFSET})) && \
           export KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://${POD_IP}:9092{{ if kindIs "string" $advertisedListenersOverride }}{{ printf ",%s" $advertisedListenersOverride }}{{ end }} && \
+          export KAFKA_ZOOKEEPER_CONNECT={{ include "cp-kafka.cp-zookeeper.service-name" . | quote }} && \
```

Comment:
Despite the yaml being correctly built by helm, the kubernetes api merges it so that the old value of `KAFKA_ZOOKEEPER_CONNECT` sticks around. The helpers partial makes sure to incorporate all possible values into `cp-kafka.cp-zookeeper.service-name` so the easiest way to fix what I was seeing was to pass it in here instead of as a pod env variable.